ubuntu/R/Dockerfile

```diff
@@ -38,8 +44,6 @@ COPY Rprofile.site /usr/lib/R/etc/Rprofile.site
 
 # Rstudio installation.
 RUN apt-get update \
```

Comment:
Do we need another `apt-get update`?

Reply:
Not really, but kept it as is to make it less error prune to changes in the future.